invenio_rdm_records/services/review/service.py

```diff
@@ -143,19 +145,51 @@ def delete(self, identity, id_, revision_id=None, uow=None):
         return True
 
     @unit_of_work()
-    def submit(self, identity, id_, data=None, revision_id=None, uow=None):
-        """Submit record for review."""
-        # Get record and check permission
+    def submit(
+        self, identity, id_, data=None, require_review=None, revision_id=None, uow=None
+    ):
+        """Submit record for review or direct publish to the community."""
         draft = self.draft_cls.pid.resolve(id_, registered_only=False)
-        self.require_permission(identity, "update_draft", record=draft)
-
         # Preconditions
         if draft.parent.review is None:
             raise ReviewNotFoundError()
 
         # since it is submit review action, assume the receiver is community
         resolved_community = draft.parent.review.receiver.resolve()
 
+        if require_review:
```

Comment:
suggestion: to remove one nesting level and makes things more readable, we could:
```
try:
    current_communities.service.require_permission(
                    identity, "direct_publish", record=resolved_community
                )
except...
    require_review = True

if require_review:
    ...
else:
    ...

```